httpx/_multipart.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, name: str, value: FileTypes) -> None:
     def get_length(self) -> int:
         headers = self.render_headers()
 
-        if isinstance(self.file, (str, bytes)):
```

Comment:
The low-footprint change here is presumably to keep this check the same, but use the following...

`return len(headers) + len(to_bytes(self.file))`

Right?